practices/managing-node-projects.md

```diff
@@ -118,7 +143,7 @@ While the classification of run-time and development dependencies is usually fai
 
 In these instances it is technically true that the dependency wouldn't need to be installed in production since only the built asset would be served.  However classifying it as a development dependency would mischaracterise its contribution to the application, and could lead it to be treated with less care than it deserves (including with a looser semver range [as above](#examples-1)).
 
-In an ideal world where HTTP2, modern JavaScript syntax, and ES6 modules are widely supported, there would be no need to transpile or concatenate any dependencies even for browsers.  Instead, they would simply be loaded using `import` statements and as such those dependencies would be directly served from their installed location (and as such would umabiguiously be run-time dependencies, not development dependencies). When working within a Node.js environment this is already possible through its native support for CommonJS `require` statements. 
+In an ideal world where HTTP2, modern JavaScript syntax, and ES6 modules are widely supported, there would be no need to transpile or concatenate any dependencies even for browsers.  Instead, they would simply be loaded using `import` statements and as such those dependencies would be directly served from their installed location (and as such would umabiguiously be run-time dependencies, not development dependencies). When working within a Node.js environment this is already possible through its native support for CommonJS `require` statements.
```

Comment:
>Instead, they would simply be loaded using `import` statements and as such those dependencies would be directly served from their installed location (and as such would umabiguiously be run-time dependencies, not development dependencies).

This is 2 or 3 sentences in one, and sneaking the second "as such" into parentheses doesn't trick people into not noticing ;)

"unambiguously" is misspeeled 

contractions: "they would simply be loaded"

passive voice: "when working within a node.js env..." (I'd fix this by dropping the first part entirely and starting with "In a Node.js env..."

Reply:
i have no idea what the change is there, i imagine whitespace... but will fix!

Reply:
Yeah this is all a bit chin-strokey, I will try to simplify.